examples/embeddable_examples/public/app/register_embeddable.tsx

```diff
@@ -36,17 +41,64 @@ export const RegisterEmbeddable = () => {
           Run the example embeddables by creating a dashboard, clicking <em>Add panel</em> button,
           and then selecting <em>Embeddable examples</em> group.
         </p>
+      </EuiText>
+
+      <EuiSpacer size="l" />
+
+      <EuiText>
+        <h2>Show embeddables in the Add panel menu</h2>
         <p>
           Add your own embeddables to <em>Add panel</em> menu by attaching an action to the{' '}
           <strong>ADD_PANEL_TRIGGER</strong> trigger. Notice usage of <strong>grouping</strong> to
           nest related panel types and avoid bloating <em>Add panel</em> menu. Please reach out to
           @elastic/kibana-presentation team to coordinate menu updates.
         </p>
       </EuiText>
-
+      <EuiSpacer size="s" />
       <EuiCodeBlock language="jsx" fontSize="m" paddingSize="m">
         {registerAttachActionSource}
       </EuiCodeBlock>
+
+      <EuiSpacer size="l" />
+
+      <EuiText>
+        <h2>Specify panel placement settings (optional)</h2>
+        <p>
+          Embeddable factories can specify placement settings for their embeddables. This is done by
```

Comment:
How about just "Add an entry to <strong>registerDashboardPanelPlacementSetting</strong> to configure initial dashboard placement.  Panel placement lets you configure the width, height, and placement strategy when panels get added to a dashboard. In the example below, the Field List embeddable will be added to dashboards as a narrow and tall panel."

The descriptions should help users understand the why and less of the how. Readers can read the code if they want exact implementation details.